src/t_zset.c

```diff
@@ -4070,6 +4070,7 @@ void zrandmemberWithCountCommand(client *c, long l, int withscores) {
             if (withscores)
                 addReplyDouble(c, zval.score);
         }
+        zuiClearIterator(&src);
```

Comment:
In fact, this is a bug that currently does not cause memory leaks, because `op->type = OBJ_ZSET`, In `zuiClearIterator`, we will do nothing, but I agree that `zuiInitIterator` and `zuiClearIterator` should appear in pairs.